pkg/usageStats/stats.go

```diff
@@ -541,6 +520,77 @@ func GetUsageStats(pastXhours uint64, granularity UsageStatsGranularity, orgid u
 	return resultMap, nil
 }
 
+// The data format has evolved over time:
+// - Initially, it was: bytes, eventCount, time
+// - Then, metrics were added: bytes, eventCount, metricCount, time
+// - Later, logsBytesCount and metricsBytesCount were added. However, the new format is backward compatible with the old formats.
+// The current format is: bytes, eventCount, metricCount, time, logsBytesCount, metricsBytesCount
+func parseRecord(record []string) (ReadStats, error) {
```

Comment:
`parseStatsRecord` could be a better name. (dont forget to change it in all the error statements below as well)